app/views/responses/_answer_read_only.html.erb

```diff
@@ -31,6 +31,18 @@
   <%# no need to output data-val here since same as human readable %>
   <%= content_tag(:div, excerpt_if_exists(@response, answer), class: 'ro-val') %>
 
+<% when 'image', 'annotated_image', 'signature', 'sketch' %>
+
+  <%= link_to image_tag(answer.media_object.url(:thumb)), answer.media_object.url %>
+
+<% when 'audio' %>
+
+  <%= link_to image_tag('media/audio.png'), answer.media_object.url %>
+
+<% when 'video' %>
+
+  <%= link_to image_tag('media/video.png'), answer.media_object.url %>
+
```

Comment:
This looks great. May not have been clear from issue but these should also appear in editable answer view.